source/MapPanel.cpp

```diff
@@ -116,6 +116,12 @@ MapPanel::MapPanel(PlayerInfo &player, int commodity, const System *special)
 	hoverText.SetWrapWidth(150);
 	hoverText.SetAlignment(WrappedText::LEFT);
 	
+	// Find out how far the player is able to jump. The range of the system
+	// takes priority over the range of the player's flagship.
+	double systemRange = playerSystem ? playerSystem->JumpRange() : 0.;
+	double playerRange = player.Flagship() ? player.Flagship()->JumpRange() : 0.;
+	playerJumpDistance = systemRange ? systemRange : (playerRange ? playerRange : System::DEFAULT_NEIGHBOR_DISTANCE);
```

Comment:
You can get rid of a ternary level by initializing to the constant in the member initialization section (i.e. `) : <in here> {` ) or in the header.